analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/Roslyn/LockReleasedAllPathsTest.cs

```diff
@@ -0,0 +1,88 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2021 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SonarAnalyzer.Rules.CSharp;
+using SonarAnalyzer.UnitTest.MetadataReferences;
+using SonarAnalyzer.UnitTest.TestFramework;
+
+namespace SonarAnalyzer.UnitTest.Rules
+{
+    [TestClass]
+    public class LockReleasedAllPathsTest
+    {
+        [TestMethod]
+        [TestCategory("Rule")]
+        public void LockReleasedAllPaths_Monitor_Linear_CS() =>
+            Verifier.VerifyAnalyzer(@"TestCases\SymbolicExecution\Roslyn\LockReleasedAllPaths.Monitor.Linear.cs", new LockReleasedAllPaths());
```

Comment:
As the `LockReleasedAllPaths` should be removed, `SymbolicExecutionRunner` class can be scaffolded.
```suggestion
            Verifier.VerifyAnalyzer(@"TestCases\SymbolicExecution\Roslyn\LockReleasedAllPaths.Monitor.Linear.cs", new SymbolicExecutionRunner());
```
Once #5159 is merged, it will look something like this:
```suggestion
            Verifier.VerifyAnalyzer(@"TestCases\SymbolicExecution\Roslyn\LockReleasedAllPaths.Monitor.Linear.cs", new SymbolicExecutionRunner(), LocksReleasedAllPaths.S2222);
```